templates/showroom.php

```diff
@@ -1,37 +1,45 @@
-<ul>
-  <?php while (have_rows('field_showroom_window', 'product_cat_' . $term_id)): the_row(); ?>
-  <li>
-    <div class="showroom">
-      <div class="scene">
-        <div class="image">
+<?php
+
+use Netzstrategen\WooCommerceShowroom\Plugin;
+
+$image_attr = Plugin::isLazyLoadActive() ? ' data-no-lazy="1"' : '';
+?>
+
+<div class="showroom gallerya gallerya--slider">
+  <ul class="js-gallerya-slider js-gallerya-lightbox clearfix">
```

Comment:
(also below)  There is no .clearfix style provided by this plugin so the class should be removed.